frontend/app/src/components/home/ChatMessage.svelte

```diff
@@ -602,7 +611,7 @@
                                         <div slot="text">{$_("forward")}</div>
                                     </MenuItem>
                                 {/if}
-                                {#if confirmed && groupChat && !me && !inThread && !isProposal}
+                                {#if confirmed && groupChat && !me && !inThread && !isProposal && !inert}
```

Comment:
this is all getting a bit horrible isn't it? How much of this information is available to the client library? Probably all of it which would allow us to write `client.canReplyToMessage(msg)` - but then maybe that causes reactivity problems.